README.md

```diff
@@ -39,36 +39,36 @@ $ pip install dbt-trino
-| cert                           | The full path to a certificate file for authentication with trino                                            | Optional                                                                                                |                                          |
-| session_properties             | Sets Trino session properties used in the connection                                                         | Optional                                                                                                | `query_max_run_time: 4h`                 |
-| database                       | Specify the database to build models into                                                                    | Required                                                                                                | `analytics`                              |
-| schema                         | Specify the schema to build models into. Note: it is not recommended to use upper or mixed case schema names | Required                                                                                                | `public`                                 |
-| host                           | The hostname to connect to                                                                                   | Required                                                                                                | `127.0.0.1`                              |
-| port                           | The port to connect to the host on                                                                           | Required                                                                                                | `8080`                                   |
-| threads                        | How many threads dbt should use                                                                              | Optional (default is `1`)                                                                               | `8`                                      |
-| prepared_statements_enabled    | Enable usage of Trino prepared statements (used in `dbt seed` commands)                                      | Optional (default is `true`)                                                                            | `true` or `false`                        |
-| retries                        | Configure how many times a database operation is retried when connection issues arise                        | Optional (default is `3`)                                                                               | `10`                                     |
+| Option                         | Description                                                                                                  | Required?                                                                                               | Example                          |
```

Comment:
btw, you could extract this change to a separate PR which is much easier to grasp and to merge